examples/vite-webpack-rspack/host/vite.config.js

```diff
@@ -4,6 +4,10 @@ import { defineConfig } from 'vite';
 
 const mfConfig = {
   name: 'host',
+  dts: {
+    consumeTypes: true,
+  },
+  dev: true,
```

Comment:
https://module-federation.io/configure/dev.html some relevent options for types hot reloading too